docs/releases/1.10.txt

```diff
@@ -383,6 +389,9 @@ Models
 * A proxy model may now inherit multiple proxy models that share a common
   non-abstract parent class.
 
+* ``Model.__init__`` now recognizes keyword arguments setting values of
```

Comment:
Most people probably won't care about this, but there are tons of third-party packages that implement some kind of “virtual fields,” where people usually take virtual fields to mean whatever they infer from the implementation of GFK (registering their fields in `_meta.virtual_fields` is one example). The maintainers of those packages might find this bit relevant.